billboardd/daemon.h

```diff
@@ -295,6 +303,13 @@ class Daemon : public QObject {
             context->subscribeTo(interestingProperties);
             gconfmon->subscribeTo(interestingProperties);
 
+            if (interestingProperties.contains("tklock")) {
+                QObject::connect(&tklock, SIGNAL(locked()),
```

Comment:
why would we want to update on unlock? i originally had changed(), and checked if it was locked in daemon, but then there are race conditions with locking/unlocking very quickly. i considered doing changed(bool), but then i made it simpler, and left the unlocked() signal for completeness.